src/main/scala/strawman/collection/BitOperations.scala

```diff
@@ -0,0 +1,40 @@
+package strawman
+package collection
+
+import scala.Predef.{intWrapper, longWrapper}
+import java.lang.String
+
+/** Some bit operations.
+ *
+ *  See http://www.drmaciver.com/2008/08/unsigned-comparison-in-javascala/ for
+ *  an explanation of unsignedCompare.
+ */
+private[collection] object BitOperations {
```

Comment:
I copy-pasted this file without any change but you are totally right we should definitely get rid of it if it’s not that useful. So far we are only using `unsignedCompare` (from the inner trait `Int`).

I see that in Scala collections with use these `BitOperations` in `IntMap` and `LongMap` (which are optimized `Map` implementations for `Map[Int, ?]` and `Map[Long, ?]`).